app/gamelib/src/game_state/systems/motion.rs

```diff
@@ -21,31 +24,44 @@ pub fn system_integrate_motion(state: &mut GameState) {
  * Bounce the cubes against the bounds.
  */
 pub fn system_bounce_bounds(state: &mut GameState) {
+    let boing = Audio::new(AssetName::BoingSFX, PlayMode::OneTime);
```

Comment:
Maybe move this inside the if on line 65 since it is only used when true.

Reply:
Good point, applied!